helper/consts/error.go

```diff
@@ -10,4 +10,8 @@ var (
 	// ErrStandby is returned if an operation is performed on a standby Vault.
 	// No operation is expected to succeed until active.
 	ErrStandby = errors.New("Vault is in standby mode")
+
+	// ErrVerIncompatible is returned if an operation is performed with a request
+	// coming from a core version is less than one stored in coreDataVersionPath
+	ErrVerIncompatible = errors.New("Vault data version is incompatible with client version")
```

Comment:
This will be used when checking for `c.versionCompatible` (e.g. `HandleRequest`). Yeah, the message is incorrect.